doc/api/resources/orders.rst

```diff
@@ -133,7 +135,7 @@ last_modified                         datetime                   Last modificati
 
 .. versionchanged:: 2023.8
 
-   The ``event`` attribute has been added. The organizer-level endpoint has been added.
+   The ``event`` and ``checkin_text`` attributes has been added. The organizer-level endpoint has been added.
```

Comment:
This should probably a whole new `..versionchanged` that only includes the `checkin_text` (at 2023.10) and leave the `event` to the existing 2023.8.